### PR TITLE
Edit kafka dbz scenario

### PR DESCRIPTION
**Pull request by roldanbob**
Suggested edits for the _Change data capture with Debezium_ scenario on the OpenShift Interactive Learning portal

### CODE REVIEW

File: middleware/middleware-kafka/kafka-debezium/00-intro.md [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
+Change data capture, or CDC, is a well-established software design pattern for capturing changes to tables in a database. 
+CDC captures row-level changes that occur in database tables and emits event records for those changes to a Kafka data streaming bus. 
+You can configure applications that rely on the data in particular tables to consume the change event streams for those tables.
+Consuming applications read the streams of event records in the order in which the events occurred.
 
 ### What you will learn
 
-In this scenario you will learn more about [Debezium](http://debezium.io/), a component part of [Red Hat Integration](https://www.redhat.com/en/products/integration) that provides change data capture for any of the supported databases:
+In this scenario you will learn about [Debezium](http://debezium.io/), a component of [Red Hat Integration](https://www.redhat.com/en/products/integration) that provides change data capture for the following supported databases:
```
Comment (gunnarmorling) [inline middleware/middleware-kafka/kafka-debezium/00-intro.md]:
```suggestion
In this scenario you will learn about [Debezium](https://debezium.io/), a component of [Red Hat Integration](https://www.redhat.com/en/products/integration) that provides change data capture for the following supported databases:
```

File: middleware/middleware-kafka/kafka-debezium/00-intro.md [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
+You will deploy a complete end-to-end solution that captures events from database transaction logs and makes those events available for processing by downstream consumers through an [Apache Kafka](https://kafka.apache.org/) broker.
 
 ### What is Debezium?
 
 ![Logo](../../../assets/middleware/debezium-getting-started/debezium-logo.png)
 
-[Debezium](http://debezium.io/) is a set of distributed services capture row-level changes in your databases so that your applications can see and respond to those changes.
-Debezium records all row-level changes committed to a particular database table in a dedicated message topic.
-Each application simply reads the topic(s) they are interested in, and they see all of the events in the same order in which they occurred.
+[Debezium](http://debezium.io/) is a set of distributed services that capture row-level changes in a database.
```
Comment (gunnarmorling) [inline middleware/middleware-kafka/kafka-debezium/00-intro.md]:
```suggestion
[Debezium](https://debezium.io/) is a set of distributed services that capture row-level changes in a database. 
```

Comment (gunnarmorling) [reply middleware/middleware-kafka/kafka-debezium/00-intro.md]: Should it be linked more than once actually?

File: middleware/middleware-kafka/kafka-debezium/00-intro.md [hunk 3]
```diff
@@ -1,26 +1,31 @@
...
+Debezium records the change events for each table in a database to a dedicated Kafka topic. 
+You can configure applications to read from the topics that contain data change event records for specific tables.
+The consuming applications can then respond to change events with minimal latency.
+Applications read event records from a topic in the same order in which the events occurred.
 
-Technically Debezium utilizes the [Apache Kafka](https://kafka.apache.org/) streaming platform to distribute events captured from database.
-It is a set of plug-ins for [Kafka Connect](https://kafka.apache.org/documentation/#connect) that publish messages to a [Kafka broker](https://kafka.apache.org/documentation/#uses_messaging).
+Each Debezium source connector is built as a plugin for [Kafka Connect](https://kafka.apache.org/documentation/#connect).
+A Debezium source connector captures change events from a database and uses the [Apache Kafka](https://kafka.apache.org/) streaming platform to distribute and publish the captured event records to a [Kafka broker](https://kafka.apache.org/documentation/#uses_messaging).
```
Comment (gunnarmorling) [inline middleware/middleware-kafka/kafka-debezium/00-intro.md:29]: Same question about linking more than once.

File: middleware/middleware-kafka/kafka-debezium/99-outro.md [hunk 4]
```diff
@@ -1,10 +1,12 @@
...
-But this is just a beginning of a long journey. Please take your time and look at these resources:
-* Debezium [website](http://debezium.io/)
-* Extensive Debezium [tutorial](http://debezium.io/docs/tutorial/)
-* Debezium [examples repository](https://github.com/debezium/debezium-examples)
-* How to run Debezium on [Minishift](http://debezium.io/docs/openshift/)
+But this is just the beginning.
+Continue to discover more about Debezium by exploring these additional resources:
+
+* [Debezium website](http://debezium.io/)
+* [Extended Debezium tutorial](http://debezium.io/docs/tutorial/)
```
Comment (gunnarmorling) [inline middleware/middleware-kafka/kafka-debezium/99-outro.md]:
```suggestion
* [Debezium website](https://debezium.io/)
* [Extended Debezium tutorial](https://debezium.io/docs/tutorial/)
```

File: middleware/middleware-kafka/kafka-debezium/99-outro.md [hunk 5]
```diff
@@ -1,10 +1,12 @@
...
+* [Debezium examples repository](https://github.com/debezium/debezium-examples)
+* [How to run Debezium on Minishift](http://debezium.io/docs/openshift/)
```
Comment (gunnarmorling) [inline middleware/middleware-kafka/kafka-debezium/99-outro.md:11]: This URL goes through a Redirect. Can you add the actual one instead?

**Review by gunnarmorling — COMMENTED** [review]
Some small remarks inline.

File: middleware/middleware-kafka/kafka-debezium/99-outro.md (hunk 5 above)
Comment (hguerrero) [reply middleware/middleware-kafka/kafka-debezium/99-outro.md:11]: Will do

**Review by hguerrero — APPROVED** [review]

**Pull request merged**